actions/responses.go

```diff
@@ -63,7 +66,23 @@ func ResponsesPost(c buffalo.Context) error {
 	var outcome Outcome
 	outcome.DataTypes = datatypes
 	outcome.RiskLevel = hr.Text
+	outcome.ID, err = uuid.NewV4()
+	if err != nil {
+		return c.Error(500, errors.New("Error generating outcome id"))
+	}
 	log.Println("Outcome response:", outcome)
 
+	// save responses to S3
+	// we also include the original questions and the outcome that was returned to the client
+	s := struct {
+		GeneratedOutcome  Outcome
+		ClientResponse    models.Responses
+		OriginalQuestions models.Questions
```

Comment:
why not just call this `Outcome`, `Responses`, and `Questions`? 😆

Reply:
I thought that reads better in the json that gets generated.

Reply:
you thought wrong

Reply:
🤣